docs/en/index.md

```diff
@@ -0,0 +1,83 @@
+# TOTP authenticator
+
+## Installation
+
+```bash
+composer require silverstripe/totp-authenticator
+```
+
+## Configuration
+
+### Encryption key
+
+You will need to define an environment variable named `SS_MFA_SECRET_KEY` with a random secret key, which is used
+for encrypting the TOTP secret. The authentication method will not be available for use until this is correctly defined.
+
+Please note that existing registered TOTP methods for users will not be usable on environments with different values
+for `SS_MFA_SECRET_KEY` than they were registered in.
+
+There are many ways to create a random secret key, the easiest
+is by executing a `php` command on the command line. The secret key length depends on your
+specific information security controls, but 32 characters is a good baseline.
+
+```bash
+php -r 'echo substr(base64_encode(random_bytes(32)), 0, 32) . "\n";'
+```
+
+### TOTP secret length
+
+You can also configure the length of the TOTP secret. This is the code that is displayed to users when they register
+to use TOTP, for example "alternatively, enter this code manually into your app." The default length is 16 characters.
+If you do not want to support manual code entry in your project, you may want to increase the length in order to
+increase the entropy of the TOTP secret, however removing the secret from the UI will require adjustments to the React
+components. See the [`RegisterHandler.secret_length`](api:SilverStripe\MFA\BackupCode\RegisterHandler->secret_length) configuration property.
+
+```yml
+SilverStripe\TOTP\RegisterHandler:
+  secret_length: 64
```

Comment:
These namespaces don't match - should they be the same?

Reply:
No they shouldn't.  Annoyingly there's loads of things called "RegisterHandler" in the various MFA modules.